dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -91,6 +91,20 @@ public static void assertHasOnlyErrors( ValidationReport report, ValidationCode.
         assertHasErrors( report, codes.length, codes );
     }
 
+    /**
+     * assertHasWarnings asserts the report contains only given warnings in any
+     * order.
+     *
+     * @param report import report to be asserted on
+     * @param warnings expected warnings
+     */
+    public static void assertHasOnlyWarnings( ImportReport report, Warning... warnings )
```

Comment:
Does this method mean "has only warnings" as in it has no errors? or does it mean it has only the warnings passed as parameter?